docs/pjrt/pjrt_integration.md

```diff
@@ -1,21 +1,32 @@
 # PJRT plugin integration
 
+## Background
+
 This doc focuses on the recommendations about how to integrate with PJRT, and
 how to test PJRT integration with JAX.
 
+### What is PJRT
+
+The PJRT C API provides the device compiler and runtime interface, as well as the interface between the framework and the hardware.
+
+A PJRT plugin is a Python package that contains a `.so` file which implements PJRT C APIs, as well as Python methods/setup to make it discoverable by the framework.
+
+For more examples of PJRT plugins see [PJRT Examples](examples.md).
+
 ## How to integrate with PJRT
 
-### Step 1: Implement [PJRT C API interface](https://github.com/openxla/xla/blob/71a4e6e6e4e9f0f8b8f25c07a32ad489aff19239/xla/pjrt/c/pjrt_c_api.h)
+### Step 1: Implement [PJRT C API interface](https://github.com/openxla/xla/tree/main/xla/pjrt/c/pjrt_c_api.h)
 
 **Option A**: You can implement the PJRT C API directly.
```

Comment:
nit: I don't think this should be called Option A. Everyone HAS to implement the PJRT C API, then you can optionally do B. Can we clarify that please? Thanks!

Reply:
How about the following:

> **1a.** You can implement only the PJRT C API directly.

> **1b (Optional)** If you're able to build against C++ code in the [xla repo](https://github.com/openxla/xla) (via forking or bazel), you can also implement the PJRT C++ API and use the C→C++ wrapper:

Reply:
Sure that works, thanks!